pkg/controller/cronjob/cronjob_controller_test.go

```diff
@@ -29,6 +29,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/tools/record"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
I believe these are needed because cronjob controller doesn conversion. Could you put this in a comment above the import?